larpandoracontent/LArThreeDReco/LArTwoViewMatching/TwoViewSimpleTracksTool.cc

```diff
@@ -0,0 +1,127 @@
+/**
+ *  @file   larpandoracontent/LArThreeDReco/LArShowerMatching/TwoViewSimpleTracksTool.cc
+ *
+ *  @brief  Implementation of the clear showers tool class.
+ *
+ *  $Log: $
+ */
+
+#include "Pandora/AlgorithmHeaders.h"
+
+#include "larpandoracontent/LArThreeDReco/LArTwoViewMatching/TwoViewSimpleTracksTool.h"
+
+using namespace pandora;
+
+namespace lar_content
+{
+
+TwoViewSimpleTracksTool::TwoViewSimpleTracksTool() :
+    m_minMatchedFraction(0.2f),
+    m_minMatchingScore(0.9f),
+    m_minMatchedSamplingPoints(5),
+    m_minXOverlapFraction(0.5f)
+{
+}
+
+//------------------------------------------------------------------------------------------------------------------------------------------
+
+bool TwoViewSimpleTracksTool::Run(TwoViewTransverseTracksAlgorithm *const pAlgorithm, MatrixType &overlapMatrix)
+{
+    if (PandoraContentApi::GetSettings(*pAlgorithm)->ShouldDisplayAlgorithmInfo())
+       std::cout << "----> Running Algorithm Tool: " << this->GetInstanceName() << ", " << this->GetType() << std::endl;
+
+    ProtoParticleVector protoParticleVector;
+    this->FindBestTrack(overlapMatrix, protoParticleVector);
+
+    const bool particlesMade(pAlgorithm->CreateThreeDParticles(protoParticleVector));
+    return particlesMade;
```

Comment:
Could just return the result from `CreateThreeDParticles` without assignment?

Reply:
I think the bool does help readability a bit, so would be inclined to leave as it is, but let me know if you feel strongly about this and I can change it! I am also tagging @absolution1  who made a similar comment below.

Reply:
No strong opinions on these returns. Feel free to pick 😄 Leaving them means that no action is required now. Removing them is straightforward, and makes things a little simpler. The compiler will likely produce the same output in either case, for our standard builds.

Reply:
Okay, I've changed it as per suggestion :)